generics/helpers/common_handler.js

```diff
@@ -374,7 +374,10 @@ const uploadPdfToCloud = async function (fileName, userId, folderPath) {
 				try {
 					// Set headers based on cloud storage type (e.g., Azure Blob Storage)
 					const headers = {
-						'Content-Type': 'application/pdf',
+						'Content-Type':
+							getSignedUrl.data.cloudStorage === CONSTANTS.common.GCP
```

Comment:
@Prajwal17Tunerlabs do we have set content type based on cloud. why this condition required?

Reply:
yes @VISHNUDAS-tunerlabs , if this condition is not given then it will fail for gcp, it will not give us the upload url